napari/utils/key_bindings.py

```diff
@@ -513,8 +513,7 @@ def on_key_release(self, event):
         """
         if event.key is None or (
             # on linux press down is treated as multiple press and release
-            event.native is not None
-            and event.native.isAutoRepeat()
+            event.native is not None and event.native.isAutoRepeat()
```

Comment:
know deviation: https://docs.astral.sh/ruff/formatter/black/#own-line-comments-on-expressions-dont-cause-the-expression-to-expand